service/controller/resource/drainer/create.go

```diff
@@ -222,20 +233,22 @@ func nodeIsMaster(node *v1.Node) bool {
 
 }
 
-func (r *Resource) logUnevictedPods(k8sClient kubernetes.Interface, ctx context.Context, node *v1.Node) {
+func (r *Resource) logUnevictedPods(k8sClient kubernetes.Interface, ctx context.Context, awsCluster *infrastructurev1alpha3.AWSCluster, node *v1.Node) {
 	// Get the list of pods for the specific node
 	nodePods, err := nodePods(k8sClient, ctx, node)
 
```

Comment:
check if node is master or worker and use this information to include it into logging/event